tools/ci/tc/tests/test_valid.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+import jsone
+import mock
+import pytest
+import requests
+import sys
+import yaml
+from jsonschema import validate
+
+here = os.path.dirname(__file__)
+root = os.path.abspath(os.path.join(here, "..", "..", "..", ".."))
+
+
+def data_path(filename):
+    return os.path.join(here, "..", "testdata", filename)
+
+
+@pytest.mark.xfail(sys.version_info.major == 2,
+                   reason="taskcluster library has an encoding bug")
+def test_verify_taskcluster_yml():
+    """Verify that the jsone in the .taskcluster.yml is valid"""
```

Comment:
```suggestion
    """Verify that the JSON in the .taskcluster.yml is valid"""
```

I think this is what this was supposed to say, but that doesn't quite make sense. The YAML file has YAML, not JSON?